### DISCUSSION TITLE
MY CONTRIBUTIONS ARE NOT SHOWN IN MY PROFILE

### DISCUSSION THREAD

**Discussion by josuereyes16** [community/community#125232] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have noticed that some of my recent contributions have not been adequately reflected. During the last few months, I have actively participated in several key projects and activities that I consider important to highlight in my professional profile but these are not reflected.
![Captura de pantalla 2024-05-22 094058](https://github.com/community/community/assets/98969633/7af97214-01f0-4ceb-a3f1-e28d659c2410)
![Captura de pantalla 2024-05-22 094200](https://github.com/community/community/assets/98969633/c86ff10d-5924-4623-b9d7-f0f3a66b13ee)

**Comment by davevad93** [comment, marked as answer]
Hi @josuereyes16 ,

The reason why your recent contributions aren't showing up it's because those commits aren't from your repo's `main` branch, as I can see from your second screenshot, you commited on the `feature/add task` branch, that's the reason. Only `commits` made in the the repository's default branch are counted.
